quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/CatalogLoader.java

```diff
@@ -334,11 +334,13 @@ else if (indexTableName.startsWith("\"public\".")) {
 					parser.consume();
 				}
 
-				List<String> groups = parser.consumeGroup('(', ')', ',').stream().map(String::trim).collect(Collectors.toList());
+				List<String> groups = parser.consumeGroup('(', ')', ',').stream().map(String::trim).map(CatalogLoader::removeOuterQuotes).collect(Collectors.toList());
 				// TODO: Add support for expressions. Now we only support column references.
 
 				Table table = catalog.getTable(indexTableName);
-				table.addIndex(new Index(indexName, groups, unique));
+				if (!table.containsIndex(groups)) {
+					table.addIndex(new Index(indexName, groups, unique));
+				}
```

Comment:
I ran a changelog with 50 changesets, at the end the tables all contained between 10 and 20 indexes with the same column.

Reply:
Was the name different every time?